### PR TITLE
Up performance simpler

### PR DESCRIPTION
**Pull request by diocas**
This is a fork of the #5986 PR, removing the last commit which might need more discussion.
@kulmann is it ok to merge this one right away?

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/22202/66/1
The following scenarios passed on retry:
- webUIUserJourney/journey1.feature:11

**Review by kulmann — APPROVED** [review]
LGTM 👍

File: rollup.config.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ import inject from '@rollup/plugin-inject'
 const production = !process.env.ROLLUP_WATCH
 const sourcemap = process.env.SOURCE_MAP === 'true'
 const { version } = require('./package.json')
+const compilationTimestamp = new Date().getTime()
+
+const config = {
+  requirejs: {},
+  cdn: process.env.CDN === 'true'
```
Comment (pascalwengerter) [inline rollup.config.js:32]: Would've loved a changelog item or docs reference for this option, but can be added later as well:)

File: rollup.config.js [hunk 2]
```diff
@@ -25,6 +25,15 @@ import inject from '@rollup/plugin-inject'
...
+}
+if (process.env.REQUIRE_TIMEOUT) {
```
Comment (pascalwengerter) [inline rollup.config.js:34]: Same here regarding changelog/documentation

**Pull request merged**